zephyr/src/vendor_specific_dut.c

```diff
@@ -23,329 +23,68 @@
 #include "vendor_specific.h"
 #include "utils.h"
 
-#ifdef HOSTAPD_SUPPORT_MBSSID_WAR
```

Comment:
Just remove this file. In a generic code it makes sense to call vendor specific, but this is Zephyr specific already, doesn't make sense, unless we use generic `indigo_*` files and implement `vendor_*` files for Zephyr.